grammars/javascript.cson

```diff
@@ -544,9 +546,44 @@
     ]
   }
   {
-    'match': '!=|!==|<=|>=|<<=|>>=|>>>=|\\*=|(?<!\\()/=|%=|\\+=|\\-=|&=|\\^=|!|%|&|\\*|/|\\-\\-|\\-|\\+\\+|\\+|~|===|==|=|<>|<|>|!|&&|\\|\\||\\?|\\:|\\^'
-    'comment': 'match 2-character operator first'
-    'name': 'keyword.operator.js'
+    'match': '%=|\\+=|\\-=|\\*=|(?<!\\()/='
+    'name': 'keyword.operator.assignment.augmented.js'
```

Comment:
`augmented` sounds really weird to me :/
Anyone have suggestions for a different name?